### ISSUE TITLE
Store the Training Data in a Standardised Format

### ISSUE THREAD

**Issue by grayJiaaoLi** [amosproj/amos2024ss08-cloud-native-llm#57]
## User story
1. As a data engineer
2. I want / need to store the training data in a standard format
3. So that the collected data are ready for trainning

## Acceptance criteria
* Store all data as a training set in a standardized format
* Ensure the standard format meets the requirements of Gemma model

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
